README.md

```diff
@@ -1 +1,105 @@
-# api-bookstore
\ No newline at end of file
+# BookStore
+The Bookstore is a website that enable users to:
+* Display a list of books.
+* Add a book.
+* Remove a selected book.
+By building this application, I will learn how to use React and Redux.
+
+## Technologies Used
+* Languages (JSX, CSS)
+* Library (React)
+* Git
+* Gitflow
+
+## Software Requirements
+* Node.JS
+* npm
+* Code Editor
+* Git
+
+## Installation
+* Clone the repo using the command below
+
+```
+git clone git@github.com:charlesgobina/bookstore.git
+```
```

Comment:
- [ ] Let's try to be thorough in our documentation as it may be of great help to users. Your link for cloning the app is wrong. Kindly fix it. Here is the correct link: **`https://github.com/charlesgobina/api-bookstore.git`**